api.py

```diff
@@ -1,11 +1,45 @@
-
 from flask import Flask
+from alembic import op
+import sqlalchemy as sa
+import psycopg2
 
 app = Flask(__name__)
-
+con = psycopg2.connect(database="db_api_teste", user="root", password="root", host="localhost", port="5432")
+cur = con.cursor()
 @app.route('/')
 def hello():
-   return "Bem vindo"
+
+    cur.execute('''  CREATE TABLE IF NOT EXISTS usuario
+           (id     SERIAL PRIMARY KEY ,
+           name   TEXT NOT NULL,
+           document TEXT UNIQUE,
+           age     INT);
+           ''')
+    con.commit()
+
+    return "Bem vindo"
+
+@app.route("/cadastrar/<string:name>/<string:document>/<int:age>")
```

Comment:
Como você está inserindo no banco, isso deveria ser um post recebendo um body:

```suggestion
@app.route("/pessoa", methods=['POST'])
    if not request.json or not 'name' in request.json or not 'document' in request.json:
        abort(400)
```

Veja que também no endpoint eu passo o recurso, o próprio fato de ser um post já diz que eu vou criar uma pessoa